src/puppetlabs/puppetdb/dashboard.clj

```diff
@@ -28,3 +28,16 @@
                 (compojure/context (get-route this) [])
                 (add-ring-handler this))
            context)))
+
+(def dashboard-redirect
+  (app ["" &] {:get (fn [req] (rr/redirect "/pdb/dashboard/index.html"))}))
```

Comment:
no, reverse like what comidi does. If routing is url => some structured thing (like a handler fn), then reverse routing is the structured thing => url.